deployLOCAL/runETL.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  9 22:45:43 2019
+
+@author: zackkingbackup
+"""
+import sys
+sys.path.append('/Users/zackkingbackup/Documents/Job Search/CapitalOne/sample-project-data-engineer/src/')
```

Comment:
I'd prefer to use a combination of `setup.py` and `requirements.txt` to make these modules available, rather than having a `sys.path.append` with a hard-code path from your laptop ;) . 

I'll put a quick writeup on the PR of what I would like to see for `setup.py` + `requirements.txt`.

Reply:
I thought I got all the laptop paths out! Totally missed this one. Started off with just a few of them in my code while I was messing around with `boto3` trying to get various things to work locally. 

I see the `setup.py` and `requirements.txt` writeup. It makes sense right now but I'll probably still bother you with a question or two once I start putting them together.